### PR TITLE
perf(apikey): add error for apikey/register from scratch

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: packages/fx-core/src/component/driver/apiKey/create.ts [hunk 1]
```diff
@@ -170,7 +171,9 @@ export class CreateApiKeyDriver implements StepDriver {
     }
 
     if (args.primaryClientSecret && !this.validateSecret(args.primaryClientSecret)) {
-      throw new ApiKeyClientSecretInvalidError(actionName);
+      throw args.primaryClientSecret == " "
```
Comment (huimiu) [inline packages/fx-core/src/component/driver/apiKey/create.ts:174]: should we use "===" here?

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -852,6 +852,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. The length of client secret should be in this range: >=10 and <=128",
   "driver.apiKey.error.domainInvalid": "Domain is invalid. Domain for API key should follow: 1. Max %d domain(s) per API key. 2. Use comma to separate domains",
   "driver.apiKey.error.failedToGetDomain": "Unable to get domain from API specification. Make sure your API specification is valid.",
+  "driver.apiKey.error.clientSecretFromScratchInvalid": "Client secret is invalid. If you are working on an new Open API spec, please refer to the ReadMe for more detail",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:855]: Proposed: Invalid client secret. If you are working on an new Open API spec, refer the ReadMe file for details.

File: packages/fx-core/resource/package.nls.json [hunk 3]
```diff
@@ -852,6 +852,7 @@
   "driver.apiKey.error.clientSecretInvalid": "Client secret is invalid. The length of client secret should be in this range: >=10 and <=128",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:852]: Proposed: Invalid client secret. It should be 10 to 128 characters long.

File: packages/fx-core/resource/package.nls.json [hunk 4]
```diff
@@ -852,6 +852,7 @@
...
   "driver.apiKey.error.domainInvalid": "Domain is invalid. Domain for API key should follow: 1. Max %d domain(s) per API key. 2. Use comma to separate domains",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:853]: Proposed: Invalid domain. Please follow these rules: 1. Max %d domain(s) per API key. 2. Use comma to separate domains.

Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: Updated, thanks!

File: packages/fx-core/resource/package.nls.json (hunk 3 above)
Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: Updated, thanks!

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: Updated, thanks!

Comment (MuyangAmigo) [reply packages/fx-core/resource/package.nls.json]: `If you are working on an new Open API spec` What does this mean?

**Review by SLdragon — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: @MuyangAmigo I mean the from scratch template.

Comment (MuyangAmigo) [reply packages/fx-core/resource/package.nls.json]: @KennethBWSong Then can we say: `If you start with a new API, refer to the README file for details.`

Comment (KennethBWSong) [reply packages/fx-core/resource/package.nls.json]: @MuyangAmigo  @supkasar Updated, please help to review, thanks!

**Review by supkasar — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Pull request merged**